bin/rails

```diff
@@ -2,8 +2,3 @@
 APP_PATH = File.expand_path("../config/application", __dir__)
 require_relative "../config/boot"
 require "rails/commands"
-if ENV['RAILS_ENV'] == 'test'
```

Comment:
why is this deleted? does that remove simplecov?

Reply:
This was the cause of double simplecov

Reply:
ok sounds good